2-mandatory/4-space-colonies.js

```diff
@@ -15,32 +15,40 @@
   
 */
 
-function getSettlers() {}
+
+function getSettlers(arr) {
+    return arr
+        .map((item) => item.split(" "))
```

Comment:
I don't think you need to split and join the array, just need to check the item starts with A and does not end with 'family'.
Look at string methods `startsWith` and `endsWith`